layers/cmd/valueset.go

```diff
@@ -216,6 +220,26 @@ func (vsets Valuesets) Lookup(ctx *ls.Context, req ls.ValuesetLookupRequest) (ls
 			if err := lookup(v); err != nil {
 				return ls.ValuesetLookupResponse{}, err
 			}
+		} else if v, ok := vsets.Services[id]; ok {
+			req, err := http.NewRequest("GET", v, nil)
+			if err != nil {
+				return ls.ValuesetLookupResponse{}, err
+			}
+			q := req.URL.Query()
+			q.Add("service", fmt.Sprintf("%s_valueset", id))
```

Comment:
Let's build the query: id=t1&id-t2, etc.

q["id"]=[]string{...}

Also add key/value pairs to the query. You need to escape keys/values

Query should look like this:

TableIDs: t1, t2
K-V: k1:v1, k2:v2

Query: id=t1&id=t2&k1=v1&k2=v2

If you build a url.Query, no need to escape